src/pycram/worlds/multiverse.py

```diff
@@ -0,0 +1,454 @@
+import logging
+import os
+
+import numpy as np
+from tf.transformations import quaternion_matrix
+from typing_extensions import List, Dict, Optional
+
+from .multiverse_communication.client_manager import MultiverseClientManager
+from ..datastructures.dataclasses import AxisAlignedBoundingBox, Color, ContactPointsList, ContactPoint
+from ..datastructures.enums import WorldMode, JointType, ObjectType
+from ..datastructures.pose import Pose
+from ..datastructures.world import World
+from ..description import Link, Joint
+from ..robot_description import RobotDescription
+from ..world_concepts.constraints import Constraint
+from ..world_concepts.world_object import Object
+
+
+class Multiverse(World):
+    """
+    This class implements an interface between Multiverse and PyCRAM.
+    """
+
+    _joint_type_to_position_name: Dict[JointType, str] = {
+        JointType.REVOLUTE: "joint_rvalue",
```

Comment:
mutables shouldnt be default arguments

Reply:
Changed in the new feature goal_validation, that will be added to the pull requests soonTM